### PR TITLE
Fix properties not populating particle explorer tab (RC68)

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by ctrlaltdavid** [comment]
**Test Plan**

Enable Edit > Show Lights and Particle Systems in Create mode.

1) Check that property values are filled in in the particle properties tab (right-most tab) in the Create app's dialog when:
a) You create a new particle entity from the "Create" tab. The particle properties tab should automatically open when you create the new particle entity.
b) You have no entity selected, then select an existing particle entity in the "List" tab, and then open the particle properties tab.
c) You have no entity selected, have the "Create" tab open, select a particle entity by clicking on its in-world icon, and then open the particle properties tab. ... Repeat for the "Properties" and "Grid" tabs.
d) You have no entity selected, have the particle properties tab open, and then select a particle entity by clicking on its in-world icon.
e) You a particle entity selected and the particle properties tab open, then open another tab, and then open the particle properties tab again/

2) Check that the property values are updated when you select a different particle entity that has different property values.

3) Check that the property values are cleared when you deselect a currently-selected particle entity:
a) By clicking in the sky.
b) By clicking a non-particles entity.

https://highfidelity.manuscript.com/f/cases/14564

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10241/HighFidelity-Beta-PR13266-4efdba7.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10241/HighFidelity-Beta-Interface-PR13266-4efdba7.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10241/HighFidelity-Beta-PR13266-4efdba7.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10241/HighFidelity-Beta-Interface-PR13266-4efdba7.exe

**Comment by qligmaloney** [comment]
Currently testing

**Comment by qligmaloney** [comment]
PASS @conklin94122 Particle Explorer tab of the create app populates when a particle entity is created and when a particle entity is selected. Particle explorer tab behaves well when multiple particle entities are created

File: scripts/system/particle_explorer/particleExplorerTool.js [hunk 1]
```diff
@@ -16,37 +16,55 @@ var PARTICLE_EXPLORER_HTML_URL = Script.resolvePath('particleExplorer.html');
...
+    function sendActiveParticleProperies() {
+        that.webView.emitScriptEvent(JSON.stringify({
+            messageType: "particle_settings",
+            currentProperties: that.activeParticleProperties
+        }));
     }
 
     that.webEventReceived = function(data) {
-        var data = JSON.parse(data);
+        data = JSON.parse(data);
```
Comment (thoys) [inline scripts/system/particle_explorer/particleExplorerTool.js]: hmm I don't know about this. Replacing the value of an incoming parameter. Better be safe and add an extra local jsonData variable.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10280/HighFidelity-Beta-PR13266-146f871.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10280/HighFidelity-Beta-PR13266-146f871.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10280/HighFidelity-Beta-Interface-PR13266-146f871.dmg

**Comment by qligmaloney** [comment]
Currently testing

**Comment by qligmaloney** [comment]
PASS @conklin94122 Particle explorer tab populates and reacts as expected. User is able to switch between multiple particle emitters as well as one particle emitting entity and one non-particle emitting entity

**Pull request merged**